research/metadata-form/src/index.js

```diff
@@ -733,7 +755,8 @@ function loadForm() {
 
                 // If maintenance banner is visible, add padding to metadata form header

```

Comment:
Can we do the same for the "This is a development/testing/acceptance environment!" banner?